service/controllers/messages.go

```diff
@@ -64,26 +64,56 @@ func routes(routeObject response, w http.ResponseWriter) {
 
 	message := routeObject.message
 	messageArr := strings.Split(message, " ")
-	// messageTemp := message
 	var firstPars string
-	if strings.Contains(message, " ") {
+	var secondPars string
+	var thirdPars string
+	if(len(messageArr)>=3){
```

Comment:
please maintain the linting standards! avoid using () in **if, for, switch** cases and maintain a defined spacing in each statement